alf/algorithms/rl_algorithm.py

```diff
@@ -447,6 +447,14 @@ def summarize_train(self, experience, train_info, loss_info, params):
             if len(field) == 1:
                 summary_utils.summarize_distribution("action_dist", field[0])
 
+    @alf.configurable(whitelist=["custom_summary"])
+    def summarize_play(
+            self,
+            time_step: TimeStep,
+            custom_summary: Optional[Callable[[TimeStep], None]] = None):
+        if custom_summary is not None:
+            custom_summary(time_step)
```

Comment:
I think for play or aync eval, ALF doesn't write anything for tensorboard.

Reply:
It doesn't. I just want to inject summarize function from config for eval/play, as analogous as PR #1385, maybe can do it elsewhere?

Reply:
My bad. ALF does write tensorboard summary for eval.

Reply:
I feel it makes sense to make summarize_play have same signature as summarize_rollout, i.e., accepting Experience instead of TimeStep. Also please add doc-string.

Reply:
Updated.